### DISCUSSION TITLE
2022/10/30/ai-%E7%94%9F%E6%88%90%E7%9A%84%E4%B8%80%E4%BA%9B%E5%9B%BE%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#361] [Announcements]
# 2022/10/30/ai-%E7%94%9F%E6%88%90%E7%9A%84%E4%B8%80%E4%BA%9B%E5%9B%BE%E7%89%87

这几天在电脑上运行了 Stable Diffusion 玩了玩。这是我正的测试页面：https://qizhen.xyz/genimg

https://blog.qizhen.xyz/2022/10/30/ai-%E7%94%9F%E6%88%90%E7%9A%84%E4%B8%80%E4%BA%9B%E5%9B%BE%E7%89%87

**Comment by ruanqizhen** [comment]
ruanqizhen
2022年11月2日 09:49 编辑
看到别人一些帖子，会加入精美的照片吸引读者。我以前没法做同样的事，因为没有图片资源，网上找来，总有版权的风险，尤其是照片里还有人物的话。现在好了，自己可以创造一些照片了，完全没了版权的担忧。

回复
